client/src/components/Workflow/InvocationReport.vue

```diff
@@ -60,6 +61,9 @@ export default {
         onEdit() {
             window.location = withPrefix(`/pages/create?invocation_id=${this.invocationId}`);
         },
+        stsUrl(config) {
+            return `${this.dataUrl}/prepare_download`;
```

Comment:
I think this endpoint only works if Celery is enabled in the instance, otherwise, the request will fail.

Reply:
yeah, there is a fallback url which is used otherwise